### ISSUE TITLE
Support Xiaomi Mi Band 4

### ISSUE THREAD

**Issue by Gompman** [Sublimis/UrbanBiker#67]
Could you please add Support for Xiaomi Mi band 4

**Comment by JHenneberg** [comment]
It is not really possible to add because it should be treated as any other BLE device. Check if you have the firmware version (1.0.9.48) or newer mentioned in this report because before it can not work for sure: https://m.gsmarena.com/you_can_now_rate_watch_faces_and_share_heart_rate_data_on_your_mi_band_4-news-42930.php

**Comment by momo2xlc** [comment]
Me using "notify and fitness" in background wich allows to share mii band 4 heart rate with other apps

**Comment by JHenneberg** [comment]
Then you should be able to use it in urban biker too. I have no problems with Xiaomi band 3. But as I mentioned earlier they have to provide the standard interface for heart rate monitors which was deactivated before the mentioned firmware for the band 4.

**Comment by Gompman** [comment]
Yes, thx.Am 14.07.2020 21:47 schrieb "J. Henneberg" <notifications@github.com>:
Then you should be able to use it in urban biker too. I have no problems with Xiaomi band 3. But as I mentioned earlier they have to provide the standard interface for heart rate monitors which was deactivated before the mentioned firmware for the band 4.

**Comment by birgander2** [comment]
I'm using the Mi Band 4 without any problems in Urban Biker (since last summer). You need to start the MiFit app and a training, though, in order to continuously receive the heart rate.
